app/src/main/res/layout/grocery_cell.xml

```diff
@@ -2,50 +2,87 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This view doesn't look the same on my phone as the one you made in Sketch. On my phone, it takes up like 50% of the screen. I think we should try to fit in at least 5 items in the list at once.